### PR TITLE
completed task 4 - get request for a specific article number

### PR DESCRIPTION
**Pull request by Sangsterr**

### CODE REVIEW

**Comment by Harrison365** [comment]
Hi James :) 

for this ticket, you will need to test specifically for the article you are asking for. E.g. rather than any(Number) etc.

Please could you edit this test and send another PR to us.

Thanks 👍

**Comment by Harrison365** [comment]
Almost there :) 

We need you to test for all of the keys, including article Id. 

Please let us know when you are ready for another check 👍

**Comment by Harrison365** [comment]
Almost there :) 

We need you to test for all of the keys, including article Id. 

Please let us know when you are ready for another check 👍

**Comment by Harrison365** [comment]
Almost there :) 

We need you to test for all of the keys, including article Id. 

Please let us know when you are ready for another check 👍

File: __tests__/app.test.js [hunk 1]
```diff
@@ -40,14 +40,26 @@ describe('GET: /api/topics', () => {
...
         return request(app)
-            .get('/api/articles/10')
+            .get("/api/articles/1")
             .expect(200)
             .then(({ body }) => {
-                expect(body).toHaveLength(1)
-                expect(body[0].title).toBe('Seven inspirational thought leaders from Manchester UK')
-                expect(body[0].author).toBe('rogersop');
+                console.log(body[0]);
+                expect(body[0]).toEqual({
```
Comment (aloush-dev) [inline __tests__/app.test.js:42]: nice, much better testing

**Review by aloush-dev — APPROVED** [review]

**Pull request merged**